build_annoy_forest.py

```diff
@@ -0,0 +1,20 @@
+from navec import Navec
+from annoy import AnnoyIndex
+
+# load navec vector file and extract vocabulary
+path = 'parser_tool/static/js/src/data/navec_hudlit_v1_12B_500K_300d_100q.tar'
```

Comment:
Can we move the vector file to a non-public folder on the server side? For example, maybe move it to `parser_tool/data/navec_hudlit_v1_12B_500K_300d_100q.tar`. This will ensure it can only be used on the server-side. I think the only reason to put it in the `parser_tool/static/` folder is if it needs to be available/downloadable by the client-side JS.